apple/internal/apple_framework_import.bzl

```diff
@@ -180,6 +187,15 @@ regularly linked SDK frameworks, symbols from weakly linked frameworks do not ca
 are not present at runtime. Only applicable for static frameworks (i.e. `is_dynamic = False`).
 """,
         ),
+        "deps": attr.label_list(
+            doc = """
+A list of targets that are dependencies of the target being built, which will be
```

Comment:
"A list of binary import targets that are dependencies of the imported target. These dependencies will be linked in the top level binary."

Please copy this to the documentation definition in the other file

Reply:
done